rust/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+# We name the package `synapse` so that things like logging have the right
+# logging target.
+name = "synapse"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.63.0"
```

```suggestion
rust-version = "1.61.0"
```

I think we said 1.61.0 for now in #synapse-dev.

it even bit me :-)
```
error: package `synapse v0.1.0 (/tmp/xxx/synapse/rust)` cannot be built because it requires rustc 1.63.0 or newer, while the currently active rustc version is 1.61.0
```